### PR TITLE
Fix: Upgrading to R Batch SDK to 2018-12-01.8.0

### PR DESCRIPTION
**Pull request by brnleehng**
Removing custom R container image in the merge task

### CODE REVIEW

File: R/batch-api.R [hunk 1]
```diff
@@ -37,8 +37,18 @@ BatchUtilities <- R6::R6Class(
...
+          rAzureBatch::createResourceFile(filePath = envFile, httpUrl = envFileUrl)
+
+        if (is.null(resourceFiles))
+        {
+          resourceFiles <- append(resourceFiles, environmentResourceFile)
+        }
+        else {
+          resourceFiles <-
+            list(environmentResourceFile)
+        }
```
Comment (bgklein) [inline R/batch-api.R]: Shouldn't this be reversed if null set to list else append to existing list?

File: R/utility-job.R [hunk 2]
```diff
@@ -533,9 +523,7 @@ waitForTasksToComplete <-
         jobId)
       }
 
-      if (taskCounts$completed >= totalTasks &&
```
Comment (bgklein) [inline R/utility-job.R]: Counting service notoriously runs into problems. Might need to change to doing smart list calls in the future.

File: R/batch-api.R (hunk 1 above)
Comment (brnleehng) [reply R/batch-api.R]: Will fix this.

File: R/utility-job.R (hunk 2 above)
Comment (brnleehng) [reply R/utility-job.R]: I've added job state in the check

**Pull request merged**